content/PatternLibrary/patterns/02-Components/nav-social/nav-social.hbs

```diff
@@ -0,0 +1,14 @@
+<ul class="list list--horizontal nav--social">
+    <li class="nav__item">
+        <a href="#" aria-haspopup="true" aria-expanded="false"><img src="/content/PatternLibrary/logo-discord.svg" alt="Discord logo"></a>

```

Comment:
I approved this before I realised, but - I think these aria tags are unnecessary. There's no popup or dropdown, they're direct links. Will address in BE implementation directly but you might want to update the PL too :)